torchaudio/datasets/vctk.py

```diff
@@ -136,3 +166,67 @@ def __getitem__(self, n: int) -> Tuple[Tensor, int, str, str, str]:
 
     def __len__(self) -> int:
         return len(self._walker)
+
+
+class VCTK_092(Dataset):
+    """
+    Create a Dataset for VCTK 0.92, the latest version of the VCTK dataset.
+    Each item is a tuple of the form: (waveform, sample_rate, utterance, speaker_id, utterance_id)
+    Folder `p315` will be ignored due to the non-existent corresponding text files.
+    For more information about the dataset visit: https://datashare.is.ed.ac.uk/handle/10283/3443
+    """
+
+    _folder_txt = "txt"
+    _folder_audio = "wav48_silence_trimmed"
+    _ext_txt = ".txt"
+    _ext_audio = ".flac"
+    _except_folder = "p315"
+
+    def __init__(self,
+                 root: str,
+                 url: str = URL,
+                 folder_in_archive: str = FOLDER_IN_ARCHIVE_092,
```

Comment:
Can you also remove this `folder_in_archive` from the signature? I do not see a reason why this has to be a variable or the benefit of it. It's only used one place and if this is something different it is no longer a VCTK 0.92 dataset.